src/ui/public/visualize/visualization.js

```diff
@@ -102,14 +102,7 @@ uiModules
         const Visualization = $scope.vis.type.visualization;
         const visualization = new Visualization(getVisEl()[0], $scope.vis);
 
-        if (visualization.init) {
-          visualization.init().then(() => {
-            $scope.vis.initialized = true;
-            $scope.$emit('render');
-          });
-        } else {
-          $scope.vis.initialized = true;
-        }
+        $scope.vis.initialized = true;
```

Comment:
With this change it seems for me, that the `initialized` variable is rendered completely useless. Even though we might not remove it with this PR, could you please open an issue, that we check whether it is still needed at all (and possibly remove it, if it has become unnecessary).

Reply:
cf. https://github.com/elastic/kibana/issues/15337